abel/onion_bordas.py

```diff
@@ -57,7 +57,7 @@ def _init_abel(xc, yc):
     return val1, val2
 
 
-def onion_bordas_transform(IM, dr=1, direction="inverse", shift_grid=False,
+def onion_bordas_transform(IM, dr=1, direction="inverse", shift_grid=True,
```

Comment:
Note, this default increases the "transform" time. It might be better for the user to add a dummy column. Here is a comparison of the even pixel size (default O2-ANU1024.txt) and odd pixel, recentred analysis:
![image](https://user-images.githubusercontent.com/10932229/54581367-7a426c00-4a60-11e9-86ed-89e68fe58a1d.png)

Reply:
For performance, it probably is better for the user to add a column. But, since onion-bordas isn't one of the faster methods anyway, perhaps consistency with other transform methods is more important than performance?